src/views/statutory/List.vue

```diff
@@ -140,11 +155,36 @@ export default {
         this.offset += this.limit
         this.canLoadMore = response.data.data.length === this.limit
 
+        return this.axios.get(this.services['core'] + '/my_permissions')
+      }).then((response) => {
+        this.permissions = response.data.data
+        this.can.create = this.permissions.some(permission => permission.combined.endsWith('manage_event:agora') || permission.combined.endsWith('manage_event:epm'))
+        this.can.viewUnpublishedStatutoryEvents = this.permissions.some(permission => permission.combined.endsWith('global:show_unpublished:event'))
+        this.isLoading = false
+      }).catch((err) => {
+        if (this.axios.isCancel(err)) {
+          return
+        }
+
+        this.$root.showError('Could not fetch statutory events list', err)
+      })
+    },
+    fetchAlsoDraftEvents () {
```

Comment:
There is quite some duplicate code in fetchData and fetchAlsoDraftEvents. Is there no nice way to combine the fetching of the /all endpoint in fetchData? Maybe you can take a look at https://github.com/AEGEE/frontend/blob/master/src/views/core/bodies/List.vue for inspiration